### DISCUSSION TITLE
What to cache to speed up Github workflows?

### DISCUSSION THREAD

**Discussion by helpermethod** [jbangdev/jbang#1424] [Q&A] [answered]
Pulling down Maven dependencies on each build is slow and rather wasteful. I wonder which files I'd need to put in the cache to not download everything on every build?

I guess `.m2/repository` wouldn't hurt, and probably `$HOME/.jbang/cache/dependency_cache.json`?

**Comment by maxandersen** [comment, marked as answer]
Generically you'll want ~/.jbang cached. 

Also what's used in example at https://github.com/jbangdev/jbang-action
